### PR TITLE
doc: suit improvements

### PR DESCRIPTION
**Pull request by kittydepa**
Improve struct of docs and create a quick start guide for how to customize the SUIT DFU procedure.
Note: still an orphaned page, so not clearly visible in the NCS doc set yet.
Also does not include the move of the SUIT DFU docs to their "final" place.

See latest SUIT doc integration plan [here](https://nordicsemi.atlassian.net/wiki/spaces/SPG/pages/66403897/DFU+documentation+map+and+plan).

### CODE REVIEW

**Review by maciejpietras — APPROVED** [review]

File: doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_compare_other_dfu.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+:orphan:
+
+.. _ug_nrf54h20_suit_compare_other_dfu:
+
+DFU and bootloader comparison
+*****************************
+
+The Nordic Semiconductor implementation of the :ref:`SUIT procedure <ug_nrf54h20_suit_intro>` is the only support bootloader and Device Firmware Update (DFU) procedure for the nRF54H Series of System on Chips (SoC).
```
Comment (FrancescoSer) [inline doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_compare_other_dfu.rst:8]:
```suggestion
The Nordic Semiconductor implementation of the :ref:`SUIT procedure <ug_nrf54h20_suit_intro>` is the only supported bootloader and Device Firmware Update (DFU) procedure for the nRF54H Series of System on Chips (SoC).
```

Comment (FrancescoSer) [reply doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_compare_other_dfu.rst:8]: I'm guessing you meant that :)

**Review by FrancescoSer — CHANGES_REQUESTED** [review]

File: doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst [hunk 2]
```diff
@@ -9,42 +9,74 @@ How to customize the SUIT DFU process
...
-The specified Kconfig options in the sample can be used to modify the artifacts produced by the build system, customize the DFU process, and integrate the DFU solution with external build systems.
+Nordic Semiconductor provides a Software Update Internet of Things (SUIT) sample (```nrf54h_suit_sample``) which uses predefined configurations in the build system.
+The specified Kconfig options in the sample can be used to customize the DFU process and integrate the DFU solution with external build systems.
+This guide provides a comprehensive set of instructions for modifying values in the :ref:`SUIT manifest <ug_nrf54h20_suit_manifest_overview>`.
 
-.. note::
-    In order to customize the SUIT DFU process, we strongly recommend that you modify the values in the manifest that are associated with the ID values that are configured into the device.
-    This includes both the ``class-identifier`` and ``vendor-identifier`` values found in manifest(s).
+Understanding SUIT DFU
+**********************
```
Comment (richabp) [inline doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst:17]: Consider changing the subheading for the consistency with other docs and the image following the text here. Suggestions:
"Overview" or "Overview of the SUIT DFU procedure"

File: doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_qsg.rst [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+* Microsoft's |VSC| - The recommended IDE for the |NCS|.
+* |nRFVSC| - An add-on for |VSC| that allows you to develop applications for the |NCS|.
+* nRF Command Line Tools - A set of mandatory tools for working with the |NCS|.
+* Any additional requirements described in the ``nrf54h_suit_sample` sample``.
+
+Building the SUIT sample
+************************
+
+Start by building the SUIT sample:
+.. code-block:: console
```
Comment (richabp) [inline doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_qsg.rst:82]: Formatting issue:
```suggestion
Start by building the SUIT sample:

.. code-block:: console
```

File: doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_dfu.rst [hunk 4]
```diff
@@ -9,21 +9,24 @@ Device Firmware Update using SUIT
    :local:
    :depth: 2
 
-The Software Updates for Internet of Things (SUIT) is a metadata format and procedure for performing device firmware updates (DFU).
-SUIT was originally created by the `Internet Engineering Task Force <https://datatracker.ietf.org/wg/suit/about/>`__.
-Nordic Semiconductor has created an implementation of the SUIT procedure that is the newly introduced DFU procedure for the nRF54H Series of System-on-Chip (SoC) and will be the only DFU procedure compatible with the nRF54H Series.
+The Software Updates for Internet of Things (SUIT) is the Device Firmware Upgrade (DFU) and secure boot method for nRF54H Series of System on Chip.
```
Comment (richabp) [inline doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_dfu.rst:12]:
```suggestion
The Software Updates for Internet of Things (SUIT) is the Device Firmware Upgrade (DFU) and secure boot method for nRF54H Series of System-on-Chip.
```

**Review by richabp — CHANGES_REQUESTED** [review]

File: doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst (hunk 2 above)
Comment (kittydepa) [reply doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst]: Applied

**Review by richabp — APPROVED** [review]

**Pull request merged**